### ISSUE TITLE
8.1rc2 js error if a app gets disabled on settings/apps

### ISSUE THREAD

**Issue by libasys** [owncloud/core#17344]
TestEnviro:
8.1rc2

Step to reproduce:
1. goto settings apps
2. goto enabled tab
3. click on app to disable

Expected result
No errors

Actual result 
js error
```
TypeError: OC.Settings.Apps.State.apps[appId] is undefined apps.js:231:5
```

**Comment by DeepDiver1975** [comment]
confirmed - master affected as well

**Comment by DeepDiver1975** [comment]
THX @libasys for testing!

**Comment by PVince81** [comment]
Fix is here https://github.com/owncloud/core/pull/17353

**Comment by jvillafanez** [comment]
Fix confirmed in 8.1.0
